### PR TITLE
[Hub Generated] Review request for Microsoft.DocumentDB to add version stable/2020-04-01

### PR DESCRIPTION
**Pull request by victor-meng**
This is a PR generated at [OpenAPI Hub](https://aka.ms/openapihub). You can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/victor-meng/azure-rest-api-specs/dev-cosmos-db-Microsoft.DocumentDB-2020-04-01...master/).

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “**WaitForARMFeedback**” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [x] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Review by southpolesteve — APPROVED** [review]

**Comment by victor-meng** [comment]
@markcowl Could you please review this PR? Thanks!

File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-04-01/cosmos-db.json
Comment (MehaKaushik) [inline specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-04-01/cosmos-db.json:5284]: Can you please change the description in line 5275. That is outdated, we support these properties for MongoDB, Cassandra and Table resources as well. 
Second, let us not merge this until the corresponding change in RP is checked in and deployed in Prod.

**Review by MehaKaushik — CHANGES_REQUESTED** [review]
Left a minor comment.

File: specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-04-01/cosmos-db.json
Comment (victor-meng) [reply specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-04-01/cosmos-db.json:5284]: Do I change the description to "The system generated resource properties" then?

**Comment by victor-meng** [comment]
@MehaKaushik Could you please answer my questions above? Thanks!

**Comment by markcowl** [comment]
@victor-meng  This represents a breaking change in a stable API, per the information in PR comments, this will require approval from the API review board

**Comment by southpolesteve** [comment]
@markcowl Can you clarify what is breaking? This PR is to align the spec with the responses we are observing in production.

**Comment by markcowl** [comment]
@southpolesteve Anything checked into the master branch can be shipped as SDKs, so any breaking changes have to follow the breaking change policy that you checked off on above, or libraries shipped to customers will have breaking changes.
Are you saying that the service API has changed since this swagger was submitted, or was the original swagger incorrect?

**Comment by markcowl** [comment]
@southpolesteve Here is a link to the breaking change readout: https://github.com/Azure/azure-rest-api-specs/pull/10430/checks?check_run_id=1156818293

**Comment by southpolesteve** [comment]
@markcowl In this case I think the original spec was incorrect. The type for `_ts` is unspecified but it should be `number`

**Comment by markcowl** [comment]
@southpolesteve Please follow the breaking change process here: https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/43942/Policy-for-Breaking-Changes 

In this case, some languages may be allowing customers to send the correct data.  IMO, we should fix the swagger to be correct in this case, but the api review board will make the decision.

You will also need to clear the lintdiff error above, as shown in: https://aka.ms/ci-fix

**Comment by southpolesteve** [comment]
Closing this since @MehaKaushik is rolling the change into 2020-09-01 version

**Pull request closed without merging**
